glide-core/src/client/standalone_client.rs

```diff
@@ -259,6 +260,7 @@ impl StandaloneClient {
             ReadFrom::PreferReplica {
                 latest_read_replica_index,
             } => self.round_robin_read_from_replica(latest_read_replica_index),
+            ReadFrom::AZAffinity(_az) => todo!(),
```

Comment:
Why are we pushing code with `todo!` ? unlike the "sync" client, this code is actually being used.

If you really want to push it, change the behavior to a working code (i.e. a code that does not crash). For example:

```rust
ReadFrom::AZAffinity(_) => {
    logger_warn("get_connection", "AZAffinity is not yet supported for Standalone client, choosing a random replication");
    // pick random replica
    let replica_index = choose_random_replica_index();
    self.round_robin_read_from_replica(replica_index)
}
```